presto-main/src/main/java/io/prestosql/execution/SqlTaskExecution.java

```diff
@@ -603,6 +603,11 @@ private DriverStats getDriverStats()
         }
     }
 
+    /**
+     * Lists all plans nodes id for a noMoreSplits condition.
```

Comment:
A "noMoreSplits condition" means nothing to me. Please, describe it better, I would do something like this:
"List all plans node identifiers that [...]."

On the [...] I will explain what this "noMoreCondition" means. As a developer that do not know Presto, I have no idea of what you are talking about, so I can't understand the method purpose.